src/ports/postgres/modules/deep_learning/madlib_keras.sql_in

```diff
@@ -1750,6 +1750,20 @@ CREATE OR REPLACE FUNCTION MADLIB_SCHEMA.madlib_keras_fit(
 $$ LANGUAGE sql VOLATILE
 m4_ifdef(`__HAS_FUNCTION_PROPERTIES__', `MODIFIES SQL DATA');
 
+CREATE OR REPLACE FUNCTION MADLIB_SCHEMA.madlib_keras_generate_mst_table(
+    model_selection_table   VARCHAR,
+    model_arch_table        VARCHAR,
+    model_arch_id_list      INTEGER[],
+    compile_params_list     VARCHAR[],
+    fit_params_list         VARCHAR[]
+) RETURNS VOID AS $$
+    PythonFunctionBodyOnly(`deep_learning', `madlib_keras_model_hopper_utilities')
+    with AOControl(False):
+        mst_generator = madlib_keras_model_hopper_utilities.GenerateMstTable(**globals())
+        mst_generator.generate()
+$$ LANGUAGE plpythonu VOLATILE
+m4_ifdef(`__HAS_FUNCTION_PROPERTIES__', `MODIFIES SQL DATA', `');
+
```

Comment:
I am wondering if this function (and other param search related functions) should be in a different SQL file itself. Param search will mostly likely be a different tab in our user docs too. `madlib_keras.sql_in` is already big enough, and adding user docs for param search will just make it way bigger.

Reply:
We created a story recently for splitting up that sql file.  So I think it may be fine to just put it there for now, and split it as a part of that story.